e2e/interchaintestv8/operator/proofs.go

```diff
@@ -19,3 +27,19 @@ func (pt SupportedProofType) String() string {
 func (pt SupportedProofType) ToOperatorArgs() []string {
 	return []string{"-p", pt.String()}
 }
+
+// GetEnvProofType returns a proof type based on the environment variable SP1_PROOF_TYPE.
+// If the variable is not set, it returns a random proof type.
+func GetEnvProofType() SupportedProofType {
```

Comment:
If we are going to use a random value here, I think it would be good to log that out here so it's easier to see which values a test was run with.

Reply:
done

Reply:
I logged it out in each setup suite